exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -242,38 +259,41 @@ fn instance_configs_from_java_array(
                 .get_field(artifact_spec_obj, "serviceSpecs", SERVICE_SPECS_FIELD_TYPE)?
                 .l()?
                 .into_inner();
-            // TODO: Avoid deploy arguments duplication after ECR-3690
```

Comment:
Is the task fixed now? @skletsun

Reply:
Yes. The core part was fixed during the genesis config extraction task, and your changes fixed the EJB part of ECR-3690.